tests/lms/views/decorators/test_h_api.py

```diff
@@ -21,9 +16,39 @@
 
 @pytest.mark.usefixtures("hapi_svc")
 class TestCreateHUser:
+    def test_it_invokes_patch_for_user_update(
+        self, create_h_user, context, pyramid_request, hapi_svc
+    ):
+
+        create_h_user(pyramid_request, mock.sentinel.jwt, context)
+
+        hapi_svc.patch.assert_called_once_with(
+            f"users/{context.h_username}", {"display_name": context.h_display_name}
+        )
+
+    def test_it_continues_to_post_if_patch_raises_HAPINotFoundError(
+        self, create_h_user, context, pyramid_request, hapi_svc
+    ):
+        # It will only invoke POST if PATCH raises HAPINotFoundError
+        hapi_svc.patch.side_effect = HAPINotFoundError("whatever")
+
+        create_h_user(pyramid_request, mock.sentinel.jwt, context)
+
+        assert hapi_svc.post.call_count == 1
```

Comment:
I think this needs to call `hapi_svc.post.assert_called_once_with(...)` so that the params that're passed to `post()` are tested too

Reply:
There's actually a test below that _does_ call `hapi_svc.post.assert_called_once_with(...)` and it looks like this test is a duplicate of that one (there's no way this test could be failing or succeeding if `test_it_creates_the_user_in_h_if_it_does_not_exist()` wasn't also failing or succeeding at the same time)

Reply:
I'll remove this test. The line of thinking of the two tests is, however, different. In this one I was trying to show the flow of things that happen during different expected sad paths. In the second, I was trying to show what data gets proxied to the POST...no biggie though, I'll kill this one.